src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Indexers.Newznab
+{
+    public static class NewznabCategoryFieldOptionsConverter
+    {
+        public static List<FieldSelectOption> GetFieldSelectOptions(List<NewznabCategory> categories)
+        {
+            // Ignore categories not relevant for Lidarr
+            var ignoreCategories = new[] { 1000, 2000, 4000, 5000, 6000, 7000 };
```

Comment:
Rework the ignoreCats for ebooks instead of for music.

Reply:
commited